### ISSUE TITLE
Implement Data layer

### ISSUE THREAD

**Issue by JVitorRetucci** [novahaus/clean-stack-generator#9]
We could implement a Data layer with its default data source folders like remote, local and mock. Each subfolder with its own directory for models.

**Comment by LeandroCesarr** [comment]
We could ask the user to choose which data implementation he wants to insert, what do you think?

**Comment by JVitorRetucci** [comment]
Great idea! Like the services choices.

**Comment by JVitorRetucci** [comment]
The 'remote' subfolder could have models and modules. For the 'local' or 'mock' it would be useful to have these directories as default, but it might be unused. Do you think we keep the directories to every subfolder?

**Comment by LeandroCesarr** [comment]
I don't think so, you can create directories according to need, as in the presentation with the ui lib, what do you think?
